pyquil/quilatom.py

```diff
@@ -502,6 +502,12 @@ def _expression_to_string(expression):
                 or expression.precedence == expression.op2.precedence
                 and expression.associates in ('right', 'both')):
             right = '(' + right + ')'
+        # If op2 is a float, it will maybe represented as a multiple
+        # of pi in right. If that is the case, then we need to take
+        # extra care to insert parens. See gh-943.
+        elif isinstance(expression.op2, float) and (
```

Comment:
```suggestion
        elif (isinstance(expression.op2, float) and
```

If it were me I would snuggle the whole expression in parens to avoid the awkward dangling open paren at the end but that's me just me I love snuggling expressions what can you do

Reply:
Copying the style from the previous condition. In future, I will snuggle real good.